x-pack/plugins/infra/common/http_api/metadata_api.ts

```diff
@@ -83,6 +83,7 @@ export const InfraMetadataInfoRT = rt.partial({
   cloud: InfraMetadataCloudRT,
   host: InfraMetadataHostRT,
   agent: InfraMetadataAgentRT,
+  '@timestamp': rt.string,
```

Comment:
Q: would it be better if the API returned `timestamp` without the `@`?

Reply:
What is the difference there - Is it because of the usage of `metadata?.info?.['@timestamp']`? Where do we want to rename it - if I change the type the response should also change the `_source` and there I should use the `@timestamp`.  So after changing this type, I need to have a different type here in the response and then map the response to the `timestamp` key using the `@timestamp` value from the response. I don't see how changing this type will fix all that, it looks more confusing to use the `timestamp` mapped to the `@timestamp` and have 2 types for the response 🤔

Reply:
A new type was added as discussed ([commit](https://github.com/elastic/kibana/pull/164858/commits/2d2086bfbb334cfe8ee6a9cd9e9a0983dfc42c2d))